src/tasks/tasks.service.js

```diff
@@ -1,28 +1,48 @@
 const { getAll, getById, createOne, updateOne, deleteOne} = require('./repository');
 
 const getTasks = async () => {
-    const tasks = await getAll();
-    return tasks; 
+    try {
+        const tasks = await getAll();
```

Comment:
in all service's methods you don't use any checks so you don't need assign values of repository method calls to variables. 
return await getAll();  - enough